pkg/controller/operators/adoption_controller.go

```diff
@@ -252,26 +252,25 @@ func (r *AdoptionReconciler) adoptComponents(ctx context.Context, csv *operators
 	return utilerrors.NewAggregate(errs)
 }
 
-func (r *AdoptionReconciler) adopt(ctx context.Context, operator *decorators.Operator, component runtime.Object) error {
+func (r *AdoptionReconciler) adopt(ctx context.Context, operator *decorators.Operator, component client.Object) error {
 	m, err := meta.Accessor(component)
 	if err != nil {
 		return nil
 	}
```

Comment:
There's no need to use `m` and `meta.Accessor` on `client.Object` instances. The `client.Object` interface includes the `metav1.Object` interface.

Reply:
Thanks @joelanford, I will remove that section.